commit 8d38846690a3922081ad4c8a4fb3c4faf4d8016b
Author: Brian Bennett <brian.bennett@joyent.com>
Date:   2019-08-14T06:25:09-07:00 (8 weeks ago)
    
    OS-7911 root's .profile should use pkgsrc mozilla-rootcerts, if available
    Reviewed by: John Levon <john.levon@joyent.com>
    Approved by: John Levon <john.levon@joyent.com>

diff --git a/usr/src/cmd/nsadmin/dot-profile.sh b/usr/src/cmd/nsadmin/dot-profile.sh
index 906d43bae5..fa6fb19714 100644
--- a/usr/src/cmd/nsadmin/dot-profile.sh
+++ b/usr/src/cmd/nsadmin/dot-profile.sh
@@ -5,10 +5,15 @@
 #
 
 #
-# Copyright (c) 2018, Joyent, Inc.
+# Copyright 2019 Joyent, Inc.
 #
 
 PATH=/usr/bin:/usr/sbin:/smartdc/bin:/opt/smartdc/bin:/opt/local/bin:/opt/local/sbin:/opt/tools/bin:/opt/tools/sbin:/opt/smartdc/agents/bin
 MANPATH=/usr/share/man:/smartdc/man:/opt/smartdc/man:/opt/local/man:/opt/tools/man
 PAGER=less
-export PATH MANPATH PAGER
+# If pkgsrc-tools is set up and the mozilla-rootcerts package is installed
+# configure the platform curl to use it.
+if [[ -f /opt/tools/share/mozilla-rootcerts/cacert.pem ]]; then
+	CURL_CA_BUNDLE=/opt/tools/share/mozilla-rootcerts/cacert.pem
+fi
+export PATH MANPATH PAGER CURL_CA_BUNDLE
